libs/search-widget/src/core/tracking.ts

```diff
@@ -0,0 +1,19 @@
+import posthog from 'posthog-js';
+
+let _kbid = '';
+
+export function initTracking(kbid: string) {
+  _kbid = kbid;
+  if (window.location.hostname !== 'localhost') {
+    posthog.init('phc_TAooetKnDVSmvK9WfqtFVUtiEjxpgnPYQ5OaTWUVuY8', { api_host: 'https://eu.posthog.com' });
```

Comment:
isn't it a bit unsafe to have our posthog key like that in an open source project?